camel/utils/commons.py

```diff
@@ -68,14 +67,13 @@ def api_key_required(func: F) -> F:
         callable: The decorated function.
 
     Raises:
-        ValueError: If the OpenAI API key is not found in the environment
-            variables.
+        ValueError: If the API key is not found.
```

Comment:
`        ValueError: If the API key is not found, either as an environment variable or directly passed.`